input/vulnerabilities/Mate7_TrustZone.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name" : "Mate7 TrustZone Exploit",
+  "CVE" : [["CVE-2015-4421", "mate7-advisory"],["CVE-2015-4422", "mate7-advisory"]],
+  "Coordinated_disclosure" : "unknown",
+  "Categories" : ["system"],
+  "Details" : [["The tzdriver and TEEOS modules of the Huawei Mate 7 have vulnerabilities which may allow malicious apps to perform denial of service attacks, or gain privileges, by gaining access to the TEE", "mate7-advisory"]],
+  "Discovered_by" : [["Di Shen (returnsme)", "mate7-advisory"]],
+  "Discovered_on" : [],
+  "Submission" : [{"by":"dcc52", "on":"2019-07-08"}],
+  "Reported_on" : [["2015-08-03", "mate7-firmware-security"]],
+  "Fixed_on" : [],
+  "Fix_released_on" : [],
```

Comment:
I feel that the date from the advisory should end up somewhere in here.

Reply:
OK yes - probably a more reliable date than the one on the blog post